UIAutomationTests/TestsKeyOIDC.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace UIAutomationTests
+{
+    [Collection("NotParallel")]
+    public class TestsKeyOIDC : IClassFixture<TestFixture>
+    {
+        TestFixture testFixture;
+        public TestsKeyOIDC(TestFixture tf)
+        {
+            testFixture = tf;
+            tf.Authenticate(TestFixture.AuthenticationMethod.KeyOIDCToken);
```

Comment:
would be nice to un-authenticate when all tests in the class are done.

Ideally would be nice to have the authentication/un-authentication in every test, so every test starts with a clean state and leaves everything the way it was after its execution.

Reply:
I fully agree. I'll alter it. It was a shortcut back then to get things going.